Buildings/Experimental/DHC/Loads/Cooling/BaseClasses/BuildingTimeSeries.mo

```diff
@@ -0,0 +1,295 @@
+  connect(disFloCoo.PPum,addPPum.u2)
+    annotation (Line(points={{141,-268},{200,-268},{200,74},{238,74}},color={0,0,127}));
+  connect(noHea.y,addPPum.u1)
+    annotation (Line(points={{152,120},{170,120},{170,86},{238,86}}, color={0,0,127}));
+  connect(noCoo.y,addPPum.u2)
+    annotation (Line(points={{152,80},{200,80},{200,74},{238,74}}, color={0,0,127}));
+  connect(noHea.y,addPFan.u1)
+    annotation (Line(points={{152,120},{180,120},{180,126},{238,126}},
+                                                                     color={0,0,127}));
+  connect(noCoo.y,addPFan.u2)
+    annotation (Line(points={{152,80},{200,80},{200,114},{238,114}},
+                                                                   color={0,0,127}));
+  connect(terUniCoo.PFan,addPFan.u2)
+    annotation (Line(points={{90.8333,46},{160,46},{160,114},{238,114}},color={0,0,127}));
+  connect(terUniHea.PFan,addPFan.u1)
+    annotation (Line(points={{90.8333,-12},{180,-12},{180,126},{238,126}},color={0,0,127}));
+  connect(loa.y[3], QReqHotWat_flow) annotation (Line(points={{-259,0},{40,0},{
+          40,-120},{320,-120}},
+                             color={0,0,127}));
+  connect(disFloCoo.port_b, mulChiWatOut[1].port_a)
+    annotation (Line(points={{140,-260},{260,-260}}, color={0,127,255}));
+  connect(disFloHea.port_b, mulHeaWatOut[1].port_a)
+    annotation (Line(points={{140,-60},{260,-60}}, color={0,127,255}));
+  connect(mulHeaWatInl[1].port_b, disFloHea.port_a)
+    annotation (Line(points={{-260,-60},{120,-60}}, color={0,127,255}));
+  connect(mulChiWatInl[1].port_b, disFloCoo.port_a)
+    annotation (Line(points={{-260,-260},{120,-260}}, color={0,127,255}));
+  connect(addPFan.y, mulPFan.u)
+    annotation (Line(points={{262,120},{268,120}}, color={0,0,127}));
+  connect(addPPum.y, mulPPum.u)
+    annotation (Line(points={{262,80},{268,80}}, color={0,0,127}));
+  connect(mulQReqCoo_flow.y, QReqCoo_flow)
+    annotation (Line(points={{293,0},{320,0}}, color={0,0,127}));
+  connect(mulQReqHea_flow.y, QReqHea_flow)
+    annotation (Line(points={{293,40},{320,40}}, color={0,0,127}));
+  connect(loa.y[1], mulQReqCoo_flow.u)
+    annotation (Line(points={{-259,0},{270,0}}, color={0,0,127}));
+  connect(loa.y[2], mulQReqHea_flow.u) annotation (Line(points={{-259,0},{260,0},
+          {260,40},{270,40}}, color={0,0,127}));
+  connect(disFloHea.QActTot_flow, mulQHea_flow.u) annotation (Line(points={{141,
+          -66},{220,-66},{220,280},{268,280}}, color={0,0,127}));
+  connect(disFloCoo.QActTot_flow, mulQCoo_flow.u) annotation (Line(points={{141,
+          -266},{224,-266},{224,240},{268,240}}, color={0,0,127}));
+    annotation (Line(points={{90.8333,-12},{180,-12},{180,126},{238,126}},color={0,0,127}),
+    Documentation(
+      info="<html>
+<p>This is a simplified building model where the space cooling load is provided as time series. </p>
+</html>",
+      revisions="<html>
+<ul>
+<li></li>
```

Comment:
Is this a new model, or copied from another place?

Reply:
Thank you! This is Antoine's first implemented model and I modified it for cooling model usage. I have added the revision note accordingly.